toolkits/ob-bi/carbon-home/deployment/siddhi-files/GetInvocationDataApp.siddhi

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * This software is the property of WSO2 LLC. and its suppliers, if any.
+ * Dissemination of any information or reproduction of any material contained
+ * herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+ * You may not alter or remove any copyright or other notice from copies of this content.
+ */
+
+@App:name("GetInvocationDataApp")
+
+@App:description('API Invocation status retrieval App for the given time frame')
+
+@Source(type = 'http-request',
+     source.id='source',
+     receiver.url='http://localhost:8007/InvocationData/StatusCheckStream',
```

Comment:
Use hostname placeholder instead of localhost

Reply:
Fixed with https://github.com/wso2/reference-implementation-consumerdatastandards-au/pull/5/commits/08ebd91e49e38893507326726e09de84d6d7735f